### PR TITLE
Feature/store page

### PR DESCRIPTION
**Pull request by potatosim**

### CODE REVIEW

**Review by leon-kn — APPROVED** [review]

File: src/components/FilterByBrand/FilterByBrand.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { getFieldCount } from 'handlers/getFieldCount';
+import { convertToQuery } from 'helpers/queryHelpers';
+import { useAppSelector, useAppDispatch } from 'hooks/reduxHooks';
+import { useEffect, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import brandsData from 'data/brands';
+import Filter from 'components/Filter/Filter';
+import styled from '@emotion/styled';
+
+const StyledWrapper = styled(Filter)(() => ({
```
Comment (TheMrCrowley) [inline src/components/FilterByBrand/FilterByBrand.tsx:13]: move to separate function

File: src/components/FilterByCategory/FilterByCategory.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { FiltersQueryNames } from 'enums/FiltersQueryNames';
+import { MenuItem } from '@mui/material';
+import categoriesData from 'data/categories';
+import { convertToQuery } from 'helpers/queryHelpers';
+import { getFieldCount } from 'handlers/getFieldCount';
+import { useSearchParams } from 'react-router-dom';
+
+import styled from '@emotion/styled';
+
+const StyledWrapper = styled(Filter)(() => ({
```
Comment (TheMrCrowley) [inline src/components/FilterByCategory/FilterByCategory.tsx:15]: move to separate function

File: src/components/FilterBySearch/FilterBySearch.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    dispatch(changeSearchValue(e.target.value));
+    dispatch(applyFilters());
+  };
+
+  return (
+    <TextField
+      label="Search"
+      value={searchValue}
+      onChange={handleChange}
+      sx={{ gridArea: 'search' }}
```
Comment (TheMrCrowley) [inline src/components/FilterBySearch/FilterBySearch.tsx:32]: try to remove sx

File: src/handlers/filtersSlice.ts [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  filteredItems: ProductItem[];
+  data: ProductItem[];
+}
+
+const initialState: FiltersState = {
+  selectedFilters: {
+    categories: [],
+    brands: [],
+    price: {
+      min: 10,
```
Comment (TheMrCrowley) [inline src/handlers/filtersSlice.ts:34]: use constants here

**Review by TheMrCrowley — APPROVED** [review]

**Pull request merged**